.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -31,9 +31,9 @@ body:
       label: What version of Home Assistant Core has the issue?
       placeholder: core-
       description: >
-        Can be found in: [Settings -> About](https://my.home-assistant.io/redirect/info/).
+        Can be found in: [Settings ⇒ System ⇒ Repairs ⇒ Three Dots in Upper Right ⇒ System information](https://my.home-assistant.io/redirect/system_health/).
 
-        [![Open your Home Assistant instance and show your Home Assistant version information.](https://my.home-assistant.io/badges/info.svg)](https://my.home-assistant.io/redirect/info/)
+        [![Open your Home Assistant instance and show health information about your system.](https://my.home-assistant.io/badges/system_health.svg)](https://my.home-assistant.io/redirect/system_health/)
```

Comment:
I don't think the text should be changed to 'health information' 'version information' is fine as that is what you've asked the user to enter.

I think I would be less likely to click the link if it said 'health information'.

Reply:
Right, makes sense. I just copied the alt from the other block, I guess that's the default text - although the page is now(?) called System Information.